src/test/app/ValidatorList_test.cpp

```diff
@@ -571,64 +571,64 @@ class ValidatorList_test : public beast::unit_test::suite
 
         std::string const siteUri = "testUpdateTrusted.test";
 
-        PublicKey emptyLocalKey;
-        ManifestCache manifests;
+        PublicKey emptyLocalKeyOuter;
+        ManifestCache manifestsOuter;
         jtx::Env env (*this);
-        auto trustedKeys = std::make_unique <ValidatorList> (
-            manifests, manifests, env.timeKeeper(), env.journal);
+        auto trustedKeysOuter = std::make_unique <ValidatorList> (
+            manifestsOuter, manifestsOuter, env.timeKeeper(), env.journal);
 
-        std::vector<std::string> cfgPublishers;
-        hash_set<NodeID> activeValidators;
+        std::vector<std::string> cfgPublishersOuter;
+        hash_set<NodeID> activeValidatorsOuter;
 
-        std::size_t const n = 40;
+        std::size_t const nConst = 40;
```

Comment:
Consider constexpr

Reply:
Fixed in 42d4b481a `Misc fixups in shadow warnings commit`